client-mobile/src/containers/ShoppingListDetailsContainer/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { useShoppingList } from "@client/api/queries/useShoppingList"
+import { ShoppingList } from "@client/components/molecules/ShoppingList"
+import { ShoppingListButtons } from "@client/components/molecules/ShoppingListButtons"
+import { ItemPageTemplate } from "@client/components/templates/ItemPageTemplate"
+import { useLocalSearchParams } from "expo-router"
+
+export const ShoppingListDetailsContainer: React.FC = () => {
```

Comment:
Rename to `ShoppingItemContainer`
```suggestion
export const ShoppingItemContainer: React.FC = () => {
```